src/main/scala/io/iohk/ethereum/blockchain/sync/fast/FastSync.scala

```diff
@@ -151,6 +151,10 @@ class FastSync(
     private val heartBeat =
       scheduler.scheduleWithFixedDelay(syncRetryInterval, syncRetryInterval * 2, self, ProcessSyncing)
 
+    private val startTime: Long = System.nanoTime()
+    private val toMinutes: Long = 6000000000L
+    private def totalMinutesTaken(): Long = (System.nanoTime() - startTime) / toMinutes
```

Comment:
not sure what the gain of using nano time here is. AFAIR nano time is considerably slower than millis so maybe it would be a good idea to switch to `currentTimeMillis`?

Reply:
The reason for using `nanoTime()` is because when it come to recording elapsed time `currentTimeMillis()` is not so accurate and `nanoTime()` is recommended instead. About the performance I found older articles showing `currentTimeMillis()` being way faster, but more recent benchmarks show `nanoTime()` being faster actually :/

Reply:
AFAIK currentTimeMillis is faster, but the question is: is this considered a high precision task? can we live with less precision? both options are fine with me

Reply:
I don't have a strong reason to keep it in `nanoTime`, so I have updated the calculation to use `currentTimeMillis` instead